glean-core/android/sdk_generator.gradle

```diff
@@ -21,7 +21,7 @@ import org.gradle.api.internal.artifacts.ArtifactAttributes
 // so that it will be shared between all libraries that use Glean.  This is
 // important because it is approximately 300MB in installed size.
 
-String GLEAN_PARSER_VERSION = "1.7.0"
+String GLEAN_PARSER_VERSION = "1.8.3"
```

Comment:
We are up to 1.9.0 now...  is that important here to be current?

Reply:
Good point. Yes, let me bump this.